File: Java/app/src/main/java/kz/sgq/fs_tiktaktoe/mvp/view/FieldView.java
package kz.sgq.fs_tiktaktoe.mvp.view;

import com.arellomobile.mvp.MvpView;

import java.util.List;

public interface FieldView extends MvpView {
    void setListBox(int i, String text);
    void setStep(String text);
    void setVictoryOne(String victory);
    void setVictoryTwo(String victory);
    void textColorEnd(List<Integer> list);
    void clearField();
    void finishActivity();
    void textColor();
}

File: Kotlin/app/src/main/java/kz/sgq/kotlintictactoe/mvp/view/FieldView.kt
package kz.sgq.kotlintictactoe.mvp.view

import com.arellomobile.mvp.MvpView

interface FieldView : MvpView{
    fun setStep(text: String)
    fun setVictoryOne(victory: String)
    fun setVictoryTwo(victory: String)
    fun setListBox(i: Int, text: String)
    fun textColorEnd(list: MutableList<Int>)
    fun clearField()
    fun finishActivity()
    fun textColor()
}